src/gleam/http/request.gleam

```diff
@@ -173,20 +173,34 @@ pub fn set_method(req: Request(body), method: Method) -> Request(body) {
   Request(..req, method: method)
 }
 
-/// A request with commonly used default values. This request can be used as
-/// an initial value and then update to create the desired request.
+/// Create a request from a provided URI string. 
+pub fn new(url: String) -> Request(String) {
```

Comment:
Oh! I forgot there was a `new` already. We'll need a different name. Perhaps `to_url`?